contrib/config

```diff
@@ -1,7 +1,52 @@
-music-path        <path to your music dir>
-podcast-path      <path to your podcasts dir>
-cache-path        <path to cache dir>
-db-path           /var/lib/gonic/gonic.db
-jukebox-enabled   false
-listen-addr       127.0.0.1:4747
-scan-interval     0
+# This is the gonic server system-wide configuration file. For other
+# administrative and per user settings go th the web UI.
+#
+# The strategy used for options in the default config shipped with Gonic is to
+# specify options with their default value where possible, and leave optional
+# ones commented. Uncommented options override the default value. Options that
+# are mandatory and need setting before first run have a placeholder for their
+# values in <angle brackets>.
+
+# Interface and port to listen on. Defaults to 0.0.0.0:4747
+listen-addr                 127.0.0.1:4747
+
+# HTTP(S) request logging
+#http-log                    true
+# URL path prefix to use if behind reverse proxy
+#proxy-prefix                <prefix>
+
+# Secure connection settings. Recommended to set up unless Gonic sits behing an
+# SSL enabled reverse proxy server. Disabled by default.
+#tls-cert                    <path to TLS certificatie>
+#tls-key                     <path to TLS private key>
+
+# Gonic's internal state database location
```

Comment:
```suggestion
# gonic's internal state database location
```

Reply:
I've actually found the project's name to be incorrectly capitalized in a few more spots in the configuration file, so I fixed all those too.